network-api/networkapi/wagtailpages/tests.py

```diff
@@ -375,33 +375,35 @@ def test_get_or_create_votes(self):
 class WagtailBuyersGuideVoteTest(APITestCase, BuyersGuideTestMixin):
 
     def test_successful_vote(self):
+        product_page = self.product_page
```

Comment:
This is basically a noop refactor, I thought I would be able to add tests in this PR, but it turns out that's much more work, so I filed #7559 to tackle that monster.